### PR TITLE
Add "make snapshot" target for building packages from develop branch

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/dashboard#442
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , @dahorak , Please review

**Comment by TimothyAsir** [comment]
@mbukatov Please review

File: Makefile [hunk 1]
```diff
@@ -27,4 +27,10 @@ srpm:   dist build-pkgs-dist
 rpm:    srpm
 	mock -r epel-7-x86_64 rebuild $(NAME)-$(VERSION)-$(RELEASE).el7.src.rpm --resultdir=. --define "dist .el7"
 
+update-release:
+	sed -i tendrl-dashboard.spec \
+	  -e "/^Release:/cRelease: $(shell date +"%m_%d_%Y_%H_%M_%S")"
```
Comment (dahorak) [inline Makefile]: I think, that the version should be in form Year_Month_Day... (to be properly sortable) and it should contain git revision hash.
Please see this comment: https://github.com/Tendrl/api/issues/209#issuecomment-314371093.

**Review by dahorak — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (TimothyAsir) [reply Makefile]: So, do you mean we need some thing like this:
("%Y_%m_%d_%H_%M_%S").$(SHORTCOMMIT)

Are you ok with shortcommit at the end of the release version?

**Comment by TimothyAsir** [comment]
@mbukatov please review

File: Makefile (hunk 1 above)
Comment (mbukatov) [reply Makefile]: I would suggest this format: `("%Y%m%dT%H%M%S").$(SHORTCOMMIT)`

**Comment by TimothyAsir** [comment]
@r0h4n , @ktdreyer , @mbukatov , @dahorak , Please review

**Comment by r0h4n** [comment]
Please send this PR on "master" branch

**Pull request closed without merging**